site/content/docs/0.0/migration.md

```diff
@@ -115,7 +115,7 @@ toc: true
   </details>
 
 - <details class="mb-2">
-    <summary><span class="badge text-bg-danger">Breaking</span> keys in <code>$utilities</code> map only available when <code>$enable-bootstrap-compatibility</code> is on:</summary>
+    <summary><span class="badge text-bg-danger">Breaking</span> keys in <code>$utilities</code> map only available when <code>$enable-bootstrap-compatibility</code> is on (<a href="{{< docsref "/extend/approach/#bootstrap-compatibility" >}}">learn more about Bootstrap compatibility</a>):</summary>
```

Comment:
Pa11y-ci doesn't seem happy with adding links into `<details><summary>`:

```
Errors in http://localhost:9001/docs/0.0/migration/:

 • Ensure interactive controls are not nested
   (https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=axeAPI)

   (html > body > div:nth-child(6) > main > div:nth-child(3) > ul:nth-child(8)
   > li:nth-child(5) > details > summary)

   <summary><span class="badge text-bg-dang...</summary>

 • Ensure interactive controls are not nested
   (https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=axeAPI)

   (html > body > div:nth-child(6) > main > div:nth-child(3) > ul:nth-child(16)
   > li:nth-child(6) > details > summary)

   <summary><span class="badge text-bg-dang...</summary>
```

Let's revert this change, then. The information is already elsewhere, so it's not that important, nevermind.

Reply:
If it doesn't pass, I'll remove this